src/Core/Content/Product/DataAbstractionLayer/ProductStreamUpdater.php

```diff
@@ -173,6 +173,8 @@ public function updateProducts(array $ids, Context $context): void
 
         $version = Uuid::fromHexToBytes(Defaults::LIVE_VERSION);
 
+        $considerInheritance = $context->considerInheritance();
```

Comment:
Can you add a testcase to the `\Shopware\Core\Content\Test\Product\DataAbstractionLayer\ProductStreamUpdaterTest` where you check that the stream mapping works as expected for variants with inherited properties?

Reply:
@keulinho we added a test per your instructions

Reply:
test looks good so far